packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -491,13 +491,17 @@ void _printPluginUpdatePubspecMessage(String pluginPath, String platformsString)
   globals.printStatus('''
 
 You need to update $pluginPath/pubspec.yaml to support $platformsString.
-For more information, see https://flutter.dev/go/developing-plugins.
+For more information, see https://flutter.dev/go/plugin-platforms.
 
 ''', emphasis: true, color: TerminalColor.red);
 }
 
-const String _kNoPlatformsArgMessage = '''
+void _printNoPluginMessage(String pluginPath) {
+    globals.printError('''
 
 Must specify at least one platform using --platforms.
```

Comment:
I think we can drop this line completely; it implies an error that blocked the project from being created, which is not the case. The next line, the explanation of how to add a platform, covers exactly the guidance we want to provide.

Reply:
The below line sounds too much like a general guidance.

For example, the user might be thinking like " Oh, some default platforms(android) have created, this message tells me what I should do if I want to add more" 

I think we want to send a stronger message telling user that there is something wrong with the current project and the user has to do something to make it work. 

`Must specify at least one platform using --platforms.` might  be too strong, we can say something like "A plugin with no platform has been created...". Then it is almost like the 1st version here: https://github.com/flutter/flutter/blob/a86dab0c401be5650c0ac6dbdb291877a8704523/packages/flutter_tools/lib/src/commands/create.dart#L41
with shorter sentences.

Reply:
In the case that they don't specify any platforms, I agree that an error message like "You've created a plugin project that doesn't yet support any platforms" is a good idea.

However, I think we should be showing the "here's how to add a new platform" message regardless of whether they provide the --platforms arg or not. I'd like them to know that they can add a new platform to their plugin project at any time.

Reply:
Sounds good. I'll update.